ZenPacks/zenoss/Microsoft/MSMQ/modeler/plugins/zenoss/winrm/MSMQQueueMap.py

```diff
@@ -35,7 +38,7 @@ def _get_queue_names(self, results):
                     q_name = raw_q_name.lower().replace(
                         'formatname:direct=os:', '')
                     if q_name not in q_names:
-                        q_names.append(q_name)
+                        q_names.append(q_name.strip('._'))
```

Comment:
You check for presence in `q_names` list unmodified `q_name`, but add stripped one (without `./_` prefixes/suffixes). In case of duplicated names like "_queue" you will have duplicates in result list. Is it intentionally?